### DISCUSSION TITLE
Is it possible to connect LDAP?

### DISCUSSION THREAD

**Discussion by Skwert** [gravitational/teleport#27265] [Q&A] [answered]
The essence is that all our user accounts are stored in IPE, and we don't want to create new users in another system (which is time-consuming).

**Comment by webvictim** [comment, marked as answer]
Teleport doesn't have a direct LDAP integration. It can use SAML, OIDC or Github integrations as a remote source of truth for users.

Something like Keycloak might work to be able to pull accounts from LDAP and put an OIDC layer in front of them.

**Reply by Skwert** [reply]
thx
